### DISCUSSION TITLE
Looking for v-debounce directive (voting)

### DISCUSSION THREAD

**Discussion by fkdp06** [primefaces/community#3267] [PrimeVue]
Hi, thanks for the wonderful framework.

It would be nice to have Primevue debounce directive instead of relaying on third parties.

Example: https://github.com/Coreusa/v-debounce

``` html
<input v-model.lazy="searchTerm" v-debounce="500" placeholder="Search for something" />

  watch: {
    searchTerm() {
      // Do something with 'searchTerm' after it debounced
    }
  },
```

Please note, I do use it a lot, not just for `InputText`, but for example in various filters like `AutoComplete` or `MultiSelect`. So I expect it to be something universal across various input components.

Also not sure if approach with `v-model.lazy` and `watch` is the only one. Would be nice to have something like that:
``` html
<InputText v-model="searchTerm" v-debounce="500" @update:modelValue="debouncedSearch()" placeholder="Search for something" />

  methods: {
    debouncedSearch() {
      // Do something with 'searchTerm' after it debounced
    }
  }
```

Use cases: 
- Text search
- DataTable filters
- Form field reloads based on other field values selected/entered

Everyone, please vote for that, if you are also interested.

**Comment by adewaleafolabi** [comment]
up voting this for visibility
